src/pages/Controllers/_controllers.scss

```diff
@@ -1,9 +1,63 @@
+@import "vanilla-framework/scss/vanilla";
+
 .controllers {
+  th {
+    padding-bottom: 6px;
+    padding-top: 6px;
```

Comment:
Why?

Reply:
The header styles by default have too much padding on the top so this makes the spacing consistent between the charts and two tables.